mlflow/store/artifact/azure_data_lake_artifact_repo.py

```diff
@@ -25,34 +25,37 @@
 def _parse_abfss_uri(uri):
     """
     Parse an ABFSS URI in the format
-    "abfss://<file_system>@<account_name>.dbfs.core.windows.net/<path>",
-    returning a tuple consisting of the filesystem, account name, and path
+    "abfss://<file_system>@<account_name>.dfs.core.[windows.net|/<path>",
```

Comment:
Is the `[` left unclosed on purpose? 
```suggestion
    "abfss://<file_system>@<account_name>.dfs.core.[windows.net|chinacloudapi.cn]/<path>",
```